web/public/js/controllers/dojos-map-controller.js

```diff
@@ -105,7 +125,14 @@ function cdDojosMapCtrl($scope, $window, $state, $stateParams, $translate, $geol
     });
   });
 
-  $scope.loadDojosList = function(){
+  $scope.loadDojosList = loadDojosList;
+
+  function loadDojosList() {
+    // If we aren't on the list page already
+    if ($stateParams.list === 'false' || !$stateParams.list) {
+      $state.go('home', { list: '' });
```

Comment:
This needs a value for the URL to change when clicking the tab. You had it working at one point, but this change breaks it. :(